docker/main/requirements-wheels.txt

```diff
@@ -28,8 +28,9 @@ norfair == 2.2.*
 setproctitle == 1.3.*
 ws4py == 0.5.*
 unidecode == 1.3.*
-onnxruntime == 1.18.*
+# OpenVino & ONNX
 openvino == 2024.1.*
+onnxruntime-openvino == 1.18.*
```

Comment:
I can't seem to be able to use devcontainers in the latest version with this change.

```txt
33.26 Collecting openvino==2024.1.* (from -r /requirements-wheels.txt (line 32))
33.29   Downloading openvino-2024.1.0-15008-cp39-cp39-manylinux_2_31_aarch64.whl.metadata (8.8 kB)
33.88 ERROR: Ignored the following versions that require a different python version: 1.14.0 Requires-Python >=3.10; 1.14.0rc1 Requires-Python >=3.10; 1.14.0rc2 Requires-Python >=3.10; 1.14.1 Requires-Python >=3.10; 2.1.0 Requires-Python >=3.10; 2.1.0rc1 Requires-Python >=3.10; 2.1.1 Requires-Python >=3.10
33.88 ERROR: Could not find a version that satisfies the requirement onnxruntime-openvino==1.18.* (from versions: none)
34.69 ERROR: No matching distribution found for onnxruntime-openvino==1.18.*
```

<img width="1801" alt="image" src="https://github.com/user-attachments/assets/50987cb1-d59b-4991-a1ce-5d4b98b41244">

Reply:
Are you using a Mac? If I had to guess it's failing because there is no arm variant

Reply:
Yes, @NickM-27 I am. For MAC users, any alternative? I guess I can just use the old code for the time being to code the fast API stuff.

Reply:
https://github.com/blakeblackshear/frigate/pull/13608

Reply:
I can confirm that it works 👍